### ISSUE TITLE
Android Version 5.1  Unable to global proxy

### ISSUE THREAD

**Issue by makesail** [schwabe/ics-openvpn#1010]
# General information
1. Android Version 5.1
2. Android  level 22
3. MI 3
 
# Description of the issue
Hello, schwabe! Recently, when using 5.1 device, abnormal connection occurred, that is, after authorization and agent, through the browser IP query, found that the address has not changed! Excuse me, whether this is normal, can have solution

**Comment by schwabe** [comment]
I have no clue what you are talking about.

**Comment by makesail** [comment]
Sorry, let me rephrase the question. The equipment in question is as follows:

1.The Android Version
Lollipop

2.Android Vendor/Custom roms
XIAOMI

3.Device
MI 3

4.Version of the app (Version number/play store Version /self-built) -0.6.35

Android API is 5.1.1. After running the APP and successfully connecting the device VPN, there is no effect. The IP of the device has not changed!

Thank you very much

**Comment by cowst** [comment]
This behavior happened to me in the first generation fire tv stick.
It would say connected, but checking the ip with the browser I would still
see my ISP.
Don't know if it has any relations to this, the comments I found on that
was that that generation of stick was ignoring the vpn.

**Comment by makesail** [comment]
May be! Expect solutions from the project team

**Comment by schwabe** [comment]
0.6.35 is almost 4 years old. Please use an up to date version and try to reproduce it.

The VPNService API in the FireTV stick is broken. It will just not work at all.

**Comment by cowst** [comment]
Confusing stuff :)

Fire tv stick generations:

   1. openvpn says it works, but it does nothing
   2. works well as far as i can tell
   3. (4K) works except for the per-application filter

**Comment by schwabe** [comment]
Yeah, I get confused because people are not always clear what fireTV generations they have and some don't wor at all and some work better.

**Comment by cowst** [comment]
I have all of them and still have trouble remembering what's what :)

**Comment by makesail** [comment]
Thank you!
